plugins/GSdx/GSdx.rc

```diff
@@ -142,6 +142,7 @@ BEGIN
     CONTROL         "Alpha",IDC_ALPHAHACK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,14,71,34,8
     CONTROL         "Alpha Stencil",IDC_ALPHASTENCIL,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,108,71,57,8
     CONTROL         "Disable Depth Emulation",IDC_TC_DEPTH,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,108,84,92,8
+    CONTROL         "CPU FB Conversion", IDC_8BITCPUFBHACK, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 14, 84, 91, 8
```

Comment:
The label needs to include 8-bits somewhere. `"CPU FB Conversion"` isn't really clear on the purpose for the end-user.